.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!--
+Before submitting an issue, please search for existing issues and
+make sure it is not a duplicate.
+-->
+
+# <PATH_TO_THE_PLUGIN>
+<!-- For example: # post-scan-actions/aws-python-promote-or-quarantine -->
+
+## Describe the bug
+
+<!-- A clear and concise description of what the bug is. -->
+
+## To Reproduce
```

Comment:
You don't start every words with capital character for other titles.
So my suggestion is only for format consistency.

```suggestion
## To reproduce
```
or
```suggestion
## How to reproduce
```